utils/linux-install-etcd-client.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
can we use a new shell script? we do not want to modify any file from APISIX project.